extension/test/e2e/credit-card-3ds-native.spec.js

```diff
@@ -167,19 +181,26 @@ describe('::creditCardPayment3dsNative::', () => {
     const creditCardNativePage = new CreditCardNativePage(browserTab, baseUrl)
     const additionalPaymentDetailsString =
       await creditCardNativePage.finish3dsNativePayment()
-    const { body: finalPayment } = await ctpClient.update(
-      ctpClient.builder.payments,
-      payment.id,
-      payment.version,
-      [
-        {
-          action: 'setCustomField',
-          name: 'submitAdditionalPaymentDetailsRequest',
-          value: additionalPaymentDetailsString,
-        },
-      ]
-    )
-
-    return finalPayment
+    let result = null
+    try {
+      result = await ctpClient.update(
+        ctpClient.builder.payments,
+        payment.id,
+        payment.version,
+        [
+          {
+            action: 'setCustomField',
+            name: 'submitAdditionalPaymentDetailsRequest',
+            value: additionalPaymentDetailsString,
+          },
+        ]
+      )
+    } catch (err) {
+      logger.error(
+        'credit-card-3ds-native::performChallengeFlow::errors:',
+        err.body.errors
```

Comment:
Why do we cherry pick stuff from the error instead of logging all the valuable informations like original request, status code, headers with correlation IDs: https://github.com/commercetools/nodejs/blob/4974bb700aa9469282f38da4e40ecbb8bba7d513/packages/sdk-middleware-http/src/http.js#L228-L234
Same in all other places.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/735/commits/4d6485425a3aaa41449dc797ebcb77b893d0e784